docs/notebooks/lens_demos/binaryclassification.ipynb

```diff
@@ -362,1709 +362,7 @@
    "outputs": [
```

Comment:
My run of this in this cell throws the following error:

`lens = cl.Lens(spec=alignment_spec,
               assessments=[FairnessAssessment()],
               model=credo_model,
               data=assessment_data
               )`

`---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/Users/office/amin/git/credoai_lens/docs/notebooks/lens_demos/binaryclassification.ipynb Cell 25' in <module>
----> [1](vscode-notebook-cell:/Users/office/amin/git/credoai_lens/docs/notebooks/lens_demos/binaryclassification.ipynb#ch0000024?line=0)[ lens = cl.Lens(spec=alignment_spec,
      ]()[2](vscode-notebook-cell:/Users/office/amin/git/credoai_lens/docs/notebooks/lens_demos/binaryclassification.ipynb#ch0000024?line=1)[                assessments=[FairnessAssessment()],
      ]()[3](vscode-notebook-cell:/Users/office/amin/git/credoai_lens/docs/notebooks/lens_demos/binaryclassification.ipynb#ch0000024?line=2)[                model=credo_model,
      ]()[4](vscode-notebook-cell:/Users/office/amin/git/credoai_lens/docs/notebooks/lens_demos/binaryclassification.ipynb#ch0000024?line=3)[                data=assessment_data
      ]()[5](vscode-notebook-cell:/Users/office/amin/git/credoai_lens/docs/notebooks/lens_demos/binaryclassification.ipynb#ch0000024?line=4)[                )

File ~/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py:109, in Lens.__init__(self, governance, spec, assessments, model, data, training_data, user_id, dev_mode, logging_level, warning_level)
    ]()[106](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=105)[ self.run_time = False
    ]()[108](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=107)[ # set up assessments
--> ]()[109](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=108)[ self.assessments = self._select_assessments(assessments)
    ]()[111](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=110)[ # set up reporter objects
    ]()[112](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=111)[ self.report = None

File ~/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py:372, in Lens._select_assessments(self, candidate_assessments)
    ]()[370](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=369)[ else:
    ]()[371](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=370)[     model = self.model
--> ]()[372](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=371)[ usable_assessments = get_usable_assessments(model, dataset, candidate_assessments)
    ]()[373](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=372)[ assessment_text = f"Automatically Selected Assessments for {dataset_type} dataset: {dataset.name}\n--" + \
    ]()[374](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=373)[     '\n--'.join(usable_assessments.keys())
    ]()[375](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/lens.py?line=374)[ logging.info(assessment_text)

File ~/amin/pythonenv2/lib/python3.9/site-packages/credoai/assessment/utils.py:38, in get_usable_assessments(credo_model, credo_data, candidate_assessments)
     ]()[36](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/assessment/utils.py?line=35)[     to_check = candidate_assessments
     ]()[37](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/assessment/utils.py?line=36)[ assessments = {}
---> ]()[38](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/assessment/utils.py?line=37)[ for name, assessment_class in to_check:
     ]()[39](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/assessment/utils.py?line=38)[     assessment = assessment_class()
     ]()[40](file:///Users/office/amin/pythonenv2/lib/python3.9/site-packages/credoai/assessment/utils.py?line=39)[     if assessment.check_requirements(credo_model, credo_data):

TypeError: cannot unpack non-iterable FairnessAssessment object]()`

Reply:
Ah, I see. I'll fix.